_query-dsl/aggregations/metric/geobounds.md

```diff
@@ -45,4 +44,184 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
  }
 }
-```
\ No newline at end of file
+```
+
+## Aggregating geoshapes
+
+To aggregate on geoshape fields, first create an index and map the `location` field as a `geo_shape`:
+
+```json
+PUT national_parks
+{
+  "mappings": {
+    "properties": {
+      "location": {
+        "type": "geo_shape"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Next, index some documents into the `national_parks` index:
+
+```json
+PUT national_parks/_doc/1
+{
+  "name": "Yellowstone National Park",
+  "location":
+  {"type": "envelope","coordinates": [ [-111.15, 45.12], [-109.83, 44.12] ]}
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT national_parks/_doc/2
+{
+  "name": "Yosemite National Park",
+  "location": 
+  {"type": "envelope","coordinates": [ [-120.23, 38.16], [-119.05, 37.45] ]}
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT national_parks/_doc/3
+{
+  "name": "Death Valley National Park",
+  "location": 
+  {"type": "envelope","coordinates": [ [-117.34, 37.01], [-116.38, 36.25] ]}
+}
+```
+{% include copy-curl.html %}
+
+You can run a `geo_bounds` aggregation on the `location` field as follows:
+
+```json
+GET national_parks/_search
+{
+  "aggregations": {
+    "grouped": {
+      "geo_bounds": {
+        "field": "location",
+        "wrap_longitude": true
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The optional `wrap_longitude` parameter specifies whether the bounding box returned by the aggregation can overlap the international date line (180&deg; meridian). If `wrap_longitude` is set to `true`, the bounding box can overlap the international date line and return a bounds where the lower-left longitude is larger than the upper-right longitude. The default value for `wrap_longitude` is `true`.
```

Comment:
"return a bounds" reads oddly to me. Do we mean "return boundaries"?

Reply:
Reworded